Makefile

```diff
@@ -198,7 +198,8 @@ BL_COMMON_SOURCES	+=	common/bl_common.c			\
 				${COMPILER_RT_SRCS}			\
 				${STDLIB_SRCS}
 
-INCLUDES		+=	-Iinclude/bl1				\
+INCLUDES		+=	-Iinclude				\
```

Comment:
Why are you adding this line?

Reply:
It will then be easier to include the header files either in DT or in drivers, like what is done in kernel for example.
The include line is:
`#include <dt-bindings/clock/plat-clks.h>`

That also mean that all those files would be in include/dt-bindings/.